test/scenarios-periodics/el92-src@fips.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Sourced from cleanup_scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-liveimg.ks.template rhel-9.2-microshift-source-isolated true
+    launch_vm host1 "rhel-9.2-microshift-source-isolated" "" "" "" "" "" "1"
```

Comment:
Is there a reason we're using "isolated" image which contains all the containers?

Reply:
i've used isolated to avoid building another source ISO. (we need rpm-based ISO)
is there a problem with it ?

Reply:
Right, we do not need containers, but we need an ISO with the source RPMs already installed. 
From the list of `image-installer` images we have, the "isolated" seems to be most suitable. Let's use it without adding another ISO build.
```
$ find test/ -name \*.image-installer
test/image-blueprints/layer1-base/group1/rhel92.image-installer
test/image-blueprints/layer1-base/group1/rhel93.image-installer
test/image-blueprints/layer1-base/group2/rhel92-microshift-previous-minor.image-installer
test/image-blueprints/layer3-periodic/group1/rhel92-source-isolated.image-installer
```